app/containers/ViewProfilePage/SettingsOfUser.jsx

```diff
@@ -1,6 +1,7 @@
 import React from 'react';
```

Comment:
Currently, settings tab is not visible for users who sign up with Scatter. After we add a referral program, the settings tab will be visible for everyone but we will be hiding some of the sections.
This section needs to be broken down into components - component per section.

Reply:
I added settings tab with only referral section for Scatter users